pdns/dnsdist.cc

```diff
@@ -735,7 +735,9 @@ static void spoofResponseFromString(DNSQuestion& dq, const string& spoofContent,
   string result;
 
   if (raw) {
-    SpoofAction sa(spoofContent);
+    std::vector<std::string> raws;
+    stringtok(raws, spoofContent, " ,");
```

Comment:
Perhaps it would be better to only split on "," and not on " "? I'm not sure it really matters, though.

Reply:
Agree, changed